index.html

```diff
@@ -65,7 +65,71 @@ <h2>
                     </div>     
                 </div>
             </section>
-            <section class="introduce"></section>
+            <section class="introduce">
```

Comment:
nits
追加で、すみません。
おそらく、各セクションはユニークなものなので、classではなくidでいいかなと思います！
id:ページに1つしか設置できない
class:ページに複数ある　というイメージです。
```suggestion
            <section id="introduce">
```